From 3a1097b63aa24309bc8c6303b66795232d988883 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Fri, 1 Aug 2014 14:59:48 +0000
Subject: [PATCH] ui: use standard uint32_t

Fix compiling with musl libc by using a standard integer type.

Signed-off-by: Natanael Copa <ncopa@alpinelinux.org>
---
 ui/ui.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ui/ui.c b/ui/ui.c
index 8dcb41d..7dc7284 100644
--- a/ui/ui.c
+++ b/ui/ui.c
@@ -337,7 +337,7 @@ get_uint(char **toks, unsigned int *val, char *errstr)
 static int
 get_ip_addr(char **toks, struct in_addr *ip_addr, char *errstr)
 {
-    u_int32_t     addr;
+    uint32_t      addr;
     unsigned char val;
     char          *str, *tmpstr, *istr;
     char          *ntok;
-- 
2.0.4


